Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM amsterdam/python:3.8-buster as app
+
+ENV PYTHONUNBUFFERED 1
+ENV CONSUL_HOST=${CONSUL_HOST:-notset}
+ENV CONSUL_PORT=${CONSUL_PORT:-8500}
+ENV DATAPUNT_API_URL=${DATAPUNT_API_URL:-https://api.data.amsterdam.nl/}
+ARG BOUWDOSSIERS_OBJECTSTORE_PASSWORD
+ENV BOUWDOSSIERS_OBJECTSTORE_PASSWORD=$BOUWDOSSIERS_OBJECTSTORE_PASSWORD
```

Comment:
If BOUWDOSSIERS_OBJECTSTORE_PASSWORD is not used in the build it can be removed here and later injected as env on runtime

Reply:
Removed but will check when deploying to acceptance